automation/run-tests.sh

```diff
@@ -319,11 +295,17 @@ while true; do
         vdsm_tests
         exit
         ;;
+    --machine)
```

Comment:
how about `--baremetal`?

Reply:
According to @tyll, "baremetal" means a real machine and we are supporting running on a VM too. IMO, both are valid so I think you should discuss this.

Reply:
Then machine is ok for me.